yt/visualization/tests/test_geo_projections.py

```diff
@@ -29,7 +29,7 @@ def setup():
 def compare(ds, field, idir, test_prefix, test_name, projection,
             decimals=12, annotate=False):
     def slice_image(filename_prefix):
-        sl = yt.SlicePlot(ds, idir, field)
+        sl = yt.SlicePlot(ds, idir, field, origin = "native")
```

Comment:
so, am I right in guessing that this change and the likes are meant to force backwards compatibility in these tests results ? (nothing against it, I just want to know if I'm reading this right)